tests/bofire/utils/test_torch_tools.py

```diff
@@ -739,7 +743,7 @@ def test_get_initial_conditions_generator(sequential: bool):
     ],
 )
 def test_constrained_objective2botorch(objective):
```

Comment:
We need here also the tests for the new objective, or?

Reply:
These are added :)